### DISCUSSION TITLE
Offlimits Cereal

### DISCUSSION THREAD

**Discussion by iamkevingreen** [ctrl-alt-del-world/midway#19] [Show and tell]
![Screen Shot 2020-12-01 at 6 20 13 PM](https://user-images.githubusercontent.com/2124358/100808292-e0704d80-3401-11eb-8030-fa276e33b853.png)

This is an example post but also real :)

I built [OffLimits Cereal](https://www.eatofflimits.com/) with this starter + some reCharge jazz. We have a custom toy builder thats' built with Sanity as the CRM for collection customer points. The site is relatively simple but fairly extensive in it's modularity given the few total pages. Offlimits was actually kind of the baseplate for the Midway build and if you find any founders/offlimits helper classnames it's probably from this build.

**Comment by juanpprieto** [comment]
Great stuff as usual Kevin 🍝

**Comment by Jore** [comment]
Really neat site. Well done!
